CMakeLists.txt

```diff
@@ -115,6 +120,16 @@ if(NOT "${QUIC_LOGGING_TYPE}" MATCHES "^(etw|lttng|stdout|)$")
     message(FATAL_ERROR "QUIC_LOGGING_TYPE must be \"etw\", \"lttng\" or \"stdout\", not \"${QUIC_LOGGING_TYPE}\"")
 endif()
 
+if (QUIC_ENABLE_SANITIZERS)
+    set(QUIC_SANITIZER_ACTIVE ON)
+    set(QUIC_ENABLE_ASAN ON)
+    set(QUIC_ENABLE_LSAN ON)
+    set(QUIC_ENABLE_UBSAN ON)
+    set(QUIC_ENABLE_EXTRA_SANITIZERS ON)
+else()
+    set(QUIC_SANITIZER_ACTIVE OFF)
```

Comment:
How is this intended to work, exactly? Is the caller supposed to be able to turn on, say, just `QUIC_ENABLE_ASAN`, which automatically turns on `QUIC_SANITIZER_ACTIVE`?

Reply:
Ah yes, Good catch!